src/__test__/projections/getProjectionStatus.test.ts

```diff
@@ -0,0 +1,88 @@
+import { createTestNode } from "@test-utils";
+
+import {
+  ABORTED,
+  ABORTING,
+  COMPLETED,
+  CREATING,
+  DELETING,
+  EventStoreDBClient,
+  FAULTED,
+  LOADED,
+  LOADING,
+  LOADING_STOPPED,
+  PREPARED,
+  PREPARING,
+  RUNNING,
+  STARTING,
+  STOPPED,
+  STOPPING,
+  UnknownError,
+} from "@eventstore/db-client";
+
+describe("getProjectionStatistics", () => {
```

Comment:
**SUGGESTION:** I think it'd be more consistent if we use the same semantics in test names as in the method name.
```suggestion
describe("getProjectionStatus", () => {
```

Reply:
Copy paste issue :D